src/internal_events/unix.rs

```diff
@@ -69,6 +74,34 @@ impl<E: std::fmt::Display> InternalEvent for UnixSocketError<'_, E> {
     }
 }
 
+#[derive(Debug)]
+pub struct UnixSocketSendError<'a, E> {
+    pub(crate) error: &'a E,
+    pub path: &'a std::path::Path,
+}
+
+impl<E: std::fmt::Display> InternalEvent for UnixSocketSendError<'_, E> {
+    fn emit(self) {
+        let reason = "Unix socket send error.";
+        error!(
+            message = reason,
+            error = %self.error,
+            path = ?self.path,
+            error_type = error_type::WRITER_FAILED,
+            stage = error_stage::SENDING,
+        );
```

Comment:
```suggestion
            stage = error_stage::SENDING,
            internal_log_rate_limit = true,
        );
```

Reply:
All errors should get rate limited yeah? I think this was missing it originally and didn't get fixed by master.

Reply:
Oh snap, good catch.
I scanned through the other files and there were other events that I didn't modify but were also missing this so I went ahead and added it.

Reply:
Noice